tools/tests/s3tests-runner.sh

```diff
@@ -279,6 +279,7 @@ _main() {
     export TMPFILE
     export OUTPUT_DIR
     export PARALLEL_HOME
+    export CONTAINER_EXTRA_PARAMS
```

Comment:
You probably need to export `LIFE_CYCLE_INTERNAL_PARAM` too, to make it work when it's not running the s3gw in a container.

Reply:
Yes. CI always uses containers, but locally you might not want to build a container before running s3tests. In that case you can just point the script at your ceph source tree and if it finds the radosgw binary, it will use that.
With the parallelization, using this script over just executing the s3tests directly would give a speed advantage.